src/Tizen.NUI/src/internal/XamlBinding/BindingExpression.cs

```diff
@@ -137,7 +137,7 @@ void ApplyCore(object sourceObject, BindableObject target, BindableProperty prop
                 {
                     if ((needsGetter && part.LastGetter == null) || (needsSetter && part.NextPart == null && part.LastSetter == null))
                     {
-                        Console.WriteLine("Binding", PropertyNotFoundErrorMessage, part.Content, current, target.GetType(), property.PropertyName);
+                        Console.WriteLine($"Binding : part.Content property not found on current, target property: target.GetType.PropertyName");
```

Comment:
I think this needs be changed like this.
`Console.WriteLine("Binding", PropertyNotFoundErrorMessage, part.Content, current, target.GetType(), property.PropertyName);`
=>
`Console.WriteLine($"Binding, {PropertyNotFoundErrorMessage}, {part.Content}, {current}, {target.GetType()}, {property.PropertyName}");`
to show proper variable's value.